R/bias.R

```diff
@@ -113,3 +113,64 @@ bias <- function(true_values, predictions) {
     return(res)
   }
 }
+
+
+
+# lower <- c(6341.000, 6329.500, 6087.014, 5703.500,
```

Comment:
Is this an example? Why does the quanitle_bias not have some roxgyen around it? (Could just not export it)